go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/cloud-provider v0.22.2
 	k8s.io/klog/v2 v2.10.0
-	k8s.io/kubernetes v1.22.2
+	k8s.io/kubernetes v1.22.4
```

Comment:
version update is needed in replace section below.

Reply:
discard this one

Reply:
Actually, I think you're right, the versions in `replace` should be updated too.

This is a manual process though :disappointed:

Reply:
what i noticed is we don't have `k8s.io/kubernetes` in the replace section. looks like we need to update all kube dependencies at once.

Reply:
This is done now.